src/crashreporter/crashui/main.qml

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
As a minor wishlist item, if the size of the dump files are large, or the internet connection is slow, it may be appreciated to include a progress bar or something to indicate that the upload is in progress. Some visible change of state after clicking submit would also be helpful for the user.

(I have not actually seen this in action, please disregard my comment I'm interpreting the QML wrong).

Reply:
UX didn't want to do anything like this, just a fire and forget sort of model.  I did mention the same thing in design talks.